docs/api-reference/next.config.js/redirects.md

```diff
@@ -98,6 +98,22 @@ module.exports = {
 }
 ```
 
+Because parentheses are used for regex path matching, parentheses used in the `source` must be escaped:
```

Comment:
Shouldn't we just update the code snippet above to escape the parentheses instead of adding a new snippet?

Reply:
I'm not sure I follow. The snippet above uses parentheses to wrap a regex. The parentheses in the second/new snippet are escaped only because they don't contain a regex but are instead part of the path we'd like to match (i.e. `/english(default)/something`).

Reply:
If you prefer, I can turn this into a small section at the bottom of the page about special characters in paths (parentheses as covered here but also for example spaces which need to be URI encoded manually in the source).